src/GraphNodeManagerViewExtension/ViewModels/GridNodeViewModel.cs

```diff
@@ -56,7 +61,11 @@ public string Name
                 name = NodeModel?.Name;
                 return name;
             }
-            internal set => name = value;
+            internal set
+            {
+                name = value;
```

Comment:
I think a good practice to save unnecessary property changed notifications is to first check if `name != value`, then go ahead with it. Only in a few cases, this does not apply but that needs to be seen on a case-by-case basis.